security.md

```diff
@@ -0,0 +1,31 @@
+## Security
+
+Microsoft takes the security of our software products and services seriously, which includes all source code repositories managed through our GitHub organizations, which include [Microsoft](https://github.com/Microsoft), [Azure](https://github.com/Azure), [DotNet](https://github.com/dotnet), [AspNet](https://github.com/aspnet), [Xamarin](https://github.com/xamarin), and [many more](https://opensource.microsoft.com/).
```

Comment:
This makes it sound like this is a Microsoft-hosted official repo. We may need to reconsider including this if the repo will be made public.

Reply:
@ams0 added this file, not sure why?

Reply:
removed it in #13